src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -9,43 +18,88 @@
 import com.ctre.phoenix6.controls.MotionMagicVoltage;
 import com.ctre.phoenix6.controls.VelocityVoltage;
 import com.ctre.phoenix6.controls.PositionVoltage;
+import edu.wpi.first.math.controller.PIDController; 
 
 import com.ctre.phoenix6.hardware.TalonFX;
+
 import com.ctre.phoenix6.configs.MotionMagicConfigs;
 import com.ctre.phoenix6.configs.TalonFXConfiguration; 
 import com.ctre.phoenix6.signals.ControlModeValue; 
 import com.ctre.phoenix6.configs.Slot0Configs;
 import edu.wpi.first.wpilibj.Counter;
-import edu.wpi.first.wpilibj.Encoder;
+//import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import com.ctre.phoenix6.StatusCode;
 import edu.wpi.first.wpilibj.Servo;
 
-public class ArmSubsystem extends SubsystemBase {
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj2.command.ProfiledPIDSubsystem;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.trajectory.TrapezoidProfile.Constraints;
+import edu.wpi.first.math.trajectory.TrapezoidProfile.State;
+
+
+public class ArmSubsystem extends ProfiledPIDSubsystem {
+
+  public static final class Arm {
+    public static final int kSlotIdx = 0;
+    public static final int kPIDLoopIdx = 0;
+    public static final int kTimeoutMs = 30;
+    public static final int ARM_MAX_ANGLE = 300;
+    public static final int ARM_MIN_ANGLE = 45;
+    public static final int ROTATION_TO_DEGREES = 360;
+    public static final double GEAR_RATIO = 118.587767088;
+    public static final double ENCODER_RAW_TO_ROTATION = 8132.;
+    public static final double ENCODER_OFFSET = 76.92165; 
+    public static final int ARM_ID = 18;
+    public static final int PIZZABOX_ID = 23;
+    public static final int SERVO_PWN_SLOT = 0;
+    public static final int ENCODER_DIO_SLOT = 0;
+    public static final int AMP_ANGLE = 0;
```

Comment:
These six angles have not been tried yet, so we left them at 0 for now. Will test when we get robot once again.